xarray/tests/test_dask.py

```diff
@@ -410,31 +396,58 @@ def test_from_dask_variable(self):
                       coords={'x': range(4)}, name='foo')
         self.assertLazyAndIdentical(self.lazy_array, a)
 
-    def test_compute_args(self):
-        a = DataArray([1, 2], name='a').chunk()
-        expected = DataArray([1, 4], name='expected')
-        b = a * a
-        # compute
-        b1 = b.compute(get=dask.multiprocessing.get)
-        assert b1._in_memory
-        assert_equal(b1, expected)
-        b2 = b.compute(get=dask.multiprocessing.get, num_workers=4)
-        assert b2._in_memory
-        assert_equal(b2, expected)
-        # load
-        b3 = b.load(get=dask.multiprocessing.get, num_workers=4)
-        assert b3._in_memory
-        assert_equal(b3, expected)
-        # persist
-        b4 = b.persist(get=dask.multiprocessing.get, num_workers=4)
-        assert b4._in_memory
-        assert_equal(b4, expected)
-
-        # dataset
-        ds = a.to_dataset()
-        ds.compute(get=dask.multiprocessing.get, num_workers=4)
-        ds.load(get=dask.multiprocessing.get, num_workers=4)
-        ds.persist(get=dask.multiprocessing.get, num_workers=4)
+
+@pytest.mark.parametrize("method", ['load', 'compute'])
+def test_dask_kwargs_variable(method):
+    x = Variable('y', da.from_array(np.arange(3), chunks=(2,)))
+    with mock.patch.object(Variable, method,
+                           return_value=np.arange(3)) as mock_method:
+        getattr(x, method)(foo='bar')
+    mock_method.assert_called_with(foo='bar')
```

Comment:
I think you can skip this first test. This verifies that `Variable.compute()` was called if you directly call `Variable.compute()`.